sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -14,21 +14,21 @@ NOOBS INSTALLATION INSTRUCTIONS
       iv. Check that the SD card you inserted matches the one selected by the Tool

```

Comment:
Maybe you want to change "Tool" -> "tool" here, for consistency? :wink:

Reply:
Thanks. Incorrect use of proper nouns is one of my bugbears. Grrr.